### ISSUE TITLE
The accuracy of test_cc1 recorded in the log during training is different from that tested using the test script

### ISSUE THREAD

**Issue by Run-19** [rainy-xu/TALL4Deepfake#19]
Hi, the accuracy of test_cc1 recorded in the log during training is different from that tested using the test script for weights alone. Why is this

**Comment by isMeStranger** [comment]
> Hi, the accuracy of test_cc1 recorded in the log during training is different from that tested using the test script for weights alone. Why is this

It is because the training uses the validation set, and the test script uses the test set

**Comment by isMeStranger** [comment]
> Hi, the accuracy of test_cc1 recorded in the log during training is different from that tested using the test script for weights alone. Why is this

Have you successfully replicated their results? The cross-dataset ones too?
